docs/6.x/changelog.md

```diff
@@ -287,6 +287,14 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.5.22 LTS (October 8th, 2019)
+
+#### Bugfixes
+
+* Fix an issue with monitoring application upgrade from 5.2.
```

Comment:
Can we continue with the (hopefully useful) tradition of linking to the actual PRs?

Reply:
I didn't know we had this tradition but it does sound useful indeed. I've added the links.